src/tests/functional/shared_test_classes/src/single_layer/dft.cpp

```diff
@@ -22,26 +34,26 @@ std::string DFTLayerTest::getTestCaseName(const testing::TestParamInfo<DFTParams
     result << "SignalSize=" << CommonTestUtils::vec2str(signalSize) << "_";
     result << "Inverse=" << (opType == ngraph::helpers::DFTOpType::INVERSE) << "_";
     result << "TargetDevice=" << targetDevice;
-    return result.str();
+    auto str = result.str();
+    replace_if(str.begin(), str.end(), invalidTestSuffixChar, '_');
```

Comment:
Is this necessary?

Reply:
Not really, seems that '.' causes some problems for parsing in Eclipse IDE, but we have the same logic for all other tests, and I don't see a big problem here, reverted.